lib/inky/inkycommands.ex

```diff
@@ -93,13 +86,12 @@ defmodule Inky.Commands do
     send_command(pins, 0x7E, 0x3B)
   end
 
-  defp set_gate(pins, packed_height) do
-    # TODO: tidy up binary creation
-    send_command(pins, 0x01, :binary.list_to_bin(packed_height ++ [0x00]))
+  defp set_gate(pins, display) do
```

Comment:
Is there a reason to change this to take display in? Now the function needs to know things about the display structure when it only uses a single integer value?